src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -107,7 +107,7 @@ public ClientBuilder(BuildContext context)
                         nextPageMethods.Add(nextPageMethod);
                     }
 
-                    if (!(method.Responses.Single().ResponseBody is ObjectResponseBody objectResponseBody))
+                    if (!(method.Responses.Single(r => r.ResponseBody != null).ResponseBody is ObjectResponseBody objectResponseBody))
```

Comment:
Is there a reason this is Single instead of First? Is there not a situation in which 2 different responses return different response bodies?

Also, line 118 has `and` in the message instead of `an`.

Reply:
> Is there not a situation in which 2 different responses return different response bodies?

Never see it.